lib/history/memory/inmemory_test.go

```diff
@@ -41,34 +42,54 @@ func (s *InMemorySuite) SetUpSuite(c *C) {
 }
 
 func (s *InMemorySuite) TestRecordStatus(c *C) {
+	ctx, cancel := context.WithTimeout(context.TODO(), testTimeout)
+	defer cancel()
+
 	var timeline history.Timeline
```

Comment:
The timeline had the interface type so that it would test that the struct implemented the interface. I guess it'd be better to test that somewhere else.